test/unit/synopsys-action/github-client-service.test.ts

```diff
@@ -25,6 +25,19 @@ beforeEach(() => {
   jest.mock('@actions/artifact')
 })
 
+test('uploadSarifReport throws error when no SARIF file found', async () => {
+  const githubClientService = new GithubClientService()
+  jest.spyOn(utility, 'checkIfPathExists').mockReturnValue(false)
+
+  try {
+    await githubClientService.uploadSarifReport('test-dir', '/')
+  } catch (error) {
+    if (error instanceof Error) {
```

Comment:
Why are we putting this condition. If error is coming as other object's instance, the expect won't work. If error instance type gets changed, we should be aware of at the time of development. I believe we expect it to be Error instance only.

Reply:
updated test case